app/controllers/api/v1/keywords_controller.rb

```diff
@@ -4,10 +4,8 @@ module Api
   module V1
     class KeywordsController < ApplicationController
       def index
-        first_keyword = Keyword.new({ id: 1, name: 'First keyword' })
-        second_keyword = Keyword.new({ id: 2, name: 'Second keyword' })
-
-        render json: KeywordSerializer.new([first_keyword, second_keyword]).serializable_hash.to_json
+        keywords = Keyword.where(user: current_user)
```

Comment:
I would recommend using the [Pundit gem](https://github.com/varvet/pundit/) because it there is a lot of chance that your next Web project requires permissions/policies management. 

On the other hand, Pundit has a very nice documentation (just the README.md file) and you can find examples of usage in the previous client projects ;-)